### DISCUSSION TITLE
Adjusting the bitrate of a Axis M1135 camera using RTSP SET_PARAMETER

### DISCUSSION THREAD

**Discussion by roxlu** [AxisCommunications/discussions#205] [VAPIX]
My goal is to change the bitrate of a livestream. From my understanding I can do this by sending RTSP SET_PARAMETER requests. I've found [this documentation][axdoc] that describes how to do this using RTSP (VAPIX) and this [datasheet][axds] shows that VAPIX is supported. I'm using gstreamer and sending a GET_PARAMETER with the body `Adjustable-Stream-Settings` as shown in the [examples in the documentation][axdoc] . Though the response contains the 451 status which means the parameter is not supported. 

I'm trying to find out why I'm receiving this 451 as the datasheet says VAPIX is supported? 

[axdoc]: https://www.axis.com/vapix-library/subjects/t10175981/section/t10185128/display?section=t10185128-t10185124 
[axds]: https://www.axis.com/dam/public/1f/92/1e/datasheet-axis-m1135-network-camera-en-US-353803.pdf

The request that is sent is:

```
RTSP request message 0x7f451affcd10
request line:
method: 'GET_PARAMETER'
uri: 'rtsp://192.168.178.174/axis-media/media.amp?adjustablelivestream=1'
version: '1.0'
headers:
Content-Type', value: 'text/parameters'
body:
Adjustable-Stream-Settings:
```

And the response:
```
RTSP response message 0x7f451affcd70
status line:
code: '451'
reason: 'Parameter Not Understood'
version: '1.0
```

**Comment by vivekatoffice** [comment]
Related link:
[Rate control APIs](https://www.axis.com/vapix-library/subjects/t10175981/section/t10140804/display?section=t10140804-t10140804)
